main.py

```diff
@@ -25,6 +31,129 @@
     print(bot.user.id)
     print("------")
 
+@bot.event
+async def on_message(msg):
+    global listen_chan
+    if msg.channel.name == listen_chan:
+        await bot.process_commands(msg)
+
+# Bot commands
+@bot.group(pass_context=True)
+async def resume(ctx):
+    if ctx.invoked_subcommand is None:
+        await bot.send_message(ctx.message.channel, error_message)
+
+@resume.group(name='submit', aliases=['add'], pass_context=True)
+async def submit(ctx, *args):
+    global db
+    if len(args) != 1:
+        await bot.send_message(ctx.message.channel, error_message)
+        return
+    if db.push_resume(ctx.message.author.id, args[0]).is_success:
+        await bot.send_message(
+            ctx.message.channel,
+            "successfully added you to the resume queue.\n" +
+            "PSA: please anonymize your resumes. You can replace your resume and keep your spot with !resume replace.")
+    else:
+        await bot.send_message(ctx.message.channel, "you already have a resume in the queue. Try !resume replace [resume].")
+    
+@resume.group(name='delete', aliases=['delete'], pass_context=True)
+async def remove(ctx, *args):
+    global db
+    if len(args) != 0:
+        await bot.send_message(ctx.message.channel, error_message)
+        return
+    if db.delete_resume(ctx.message.author.id).is_success:
```

Comment:
Do we want retry functionality?

Reply:
How many times would we want to retry? Three?